spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Sequential.scala

```diff
@@ -30,61 +30,91 @@ import scala.reflect.ClassTag
 class Sequential[T: ClassTag]
 (implicit ev: TensorNumeric[T]) extends Container[Activity, Activity, T] {
 
+  private var evaluateOnly = false
+
+  /**
+   * If evaluateOnly = true, then this sequential only works in evaluate mode, and not work in
+   * training mode
+   * @param status contral evaluateOnly
+   * @return this
+   */
+  def setEvaluateOnly(status: Boolean): this.type = {
+    evaluateOnly = true
+    this
+  }
+
   override def updateOutput(input: Activity): Activity = {
-    var i = 0
-    var result = input.asInstanceOf[Activity]
-    while (i < modules.length) {
-      result = modules(i).forward(result)
-      i += 1
-    }
+    if (!evaluateOnly || !isTraining()) {
+      var i = 0
+      var result = input.asInstanceOf[Activity]
+      while (i < modules.length) {
+        result = modules(i).forward(result)
+        i += 1
+      }
 
-    this.output = result
+      this.output = result
+    } else {
+      // evaluateOnly == true and is in training mode
+      output = input
```

Comment:
Yes, but modifying layers may not be general, since user may want to control the behavior of any layers. Or we can use a separate container to control this behavior